code/controllers/subsystem/dynamic/dynamic.dm

```diff
@@ -380,6 +380,31 @@ SUBSYSTEM_DEF(dynamic)
 			advisory_string += "Your sector's advisory level is Clown Planet! Our bike horns have picked up on a large bananium stash. Clowns show a large influx of clowns on your station. We highly advise you to slip any threats to keep Honkotrasen assets within the Banana Sector. The Department of Intelligence advises defending chemistry from any clowns that are trying to make baldium or space lube."
 			return advisory_string
 
+		if(HAS_TRAIT(SSstation, STATION_TRAIT_INTERN))
+			advisory_string += "Advisory Level: <b>(TITLE HERE)</b></center><BR>"
+			advisory_string += "(Copy/Paste the summary provided by the Threat Intelligence Office in this field. You shouldn't have any trouble with this just make sure to replace this message before hitting the send button. Also, make sure there's coffee ready for the meeting at 06:00 when you're done.)"
+			return advisory_string
+
+		if(HAS_TRAIT(SSstation, STATION_TRAIT_ION_STORMS))
+			advisory_string += "Advisory Level: <b>ERROR</b></center><BR>"
+			advisory_string += scramble_message_replace_chars("Your sector's advisory level is ERROR. An electromagnetic field has stormed through nearby surveillance equipment, causing major data loss. Partial data was recovered and showed no credible threats to Nanotrasen assets within the Spinward Sector; however, the Department of Intelligence advises maintaining high alert against potential threats due to the lack of complete data.", 35)
+			return advisory_string
+
+		if(HAS_TRAIT(SSstation, STATION_TRAIT_BLIZZARD))
+			advisory_string += "Advisory Level: <b>Ice Giant</b></center><BR>"
+			advisory_string += "The ongoing blizzard has interfered with our surveillance equipment, and we cannot provide an accurate threat summary at this time. We advise you to stay safe and avoid traversing the area around the station."
+			return advisory_string
+
+		if(HAS_TRAIT(SSstation, STATION_TRAIT_ASSISTANT_GIMMICKS))
+			advisory_string += "Advisory Level: <b>Grey Sky</b></center><BR>"
+			advisory_string += "Your sector's advisory level is Grey Sky. Our sensors detect abnormal activity among the assistants assigned to your station. We advise you to closely monitor the Tool Storage, Bridge, Tech Storage, and Brig for gathering crowds or petty thievery."
+			return advisory_string
+
+		if(HAS_TRAIT(SSstation, STATION_TRAIT_WISE_COWS))
```

Comment:
Maybe at this point it should be a `/datum/station_trait` proc? like `/datum/station_trait/proc/get_advisory_note`

Then we can just

```
for(var/datum/station_trait/trait as anyhting in shuffle(SSstation.traits))
    var/trait_string = trait.get_advisory_note()
    if(trait_string)
        advisory_string += trait_string 
        break